pages/credentials.tsx

```diff
@@ -103,6 +116,30 @@ export default function CredentailsPage() {
                   />
                 </div>
 
+                <div className="grid grid-cols-4 items-center gap-4">
+                  <Label htmlFor="pinecone-environment" className="text-right">
+                    Pinecone Environment
+                  </Label>
+                  <Input
+                    id="pinecone-environment"
+                    value={pineconeEnvironment}
+                    className="col-span-3"
```

Comment:
When I first started to use pinecone, I didn't quite get what `environment` is. Would be good to put a `placeholder="us-west1-gcp"` here

Reply:
makes sense, done